source/extensions/filters/common/ext_authz/ext_authz.h

```diff
@@ -58,6 +58,9 @@ struct Response {
   // A set of HTTP headers returned by the authorization server, will be optionally added
   // (using "addCopy") to the request to the upstream server.
   Http::HeaderVector headers_to_add;
+  // A set of HTTP headers consumed by the authorization server, will be removed
```

Comment:
Sorry, what do you mean by consumed by the authorization server?

Reply:
Consumed in the sense that the consumer of these headers (ie. who these headers are for) is the authorization server. Would you like me to rephrase this?

Reply:
After reading this once again. Seems like it is fine. But probably this "(using "remove")" is not necessary?

Reply:
Okay, I will update that.